integration/lsp.test.ts

```diff
@@ -0,0 +1,125 @@
+function make_request(data) {
```

Comment:
After seeing how this is used, this might be more properly called something like `buildRequest`

Reply:
Yeah, I like that better. Also, `camelCase` BeCaUsE jAvAsCrIpT